packages/tldraw/src/lib/ui/components/Toolbar/DefaultToolbarContent.tsx

```diff
@@ -210,6 +211,11 @@ export function FrameToolbarItem() {
 	return <ToolbarItem tool="frame" />
 }
 
+/** @public @react */
+export function EmbedToolbarItem() {
+	return <ToolbarItem tool="embed" />
+}
```

Comment:
If we were going to add a menu item like this, I would suggest we make it an "insert" type tool that combined images, embeds, and other content. I'd skip it for now and leave it as a bit of an easter egg for users who know to paste links. (The menu item is only used by < 1% of users)